src/check-defs.sh

```diff
@@ -4,6 +4,7 @@ LC_ALL=C
 export LC_ALL
 
 test -z "$srcdir" && srcdir=.
+test -z "$libs" && libs=.
```

Comment:
Humm. I prefer libs to default to .libs

Reply:
Or maybe .libs if that dir exists, otherwise .

Reply:
On #679